src/averageAges.js

```diff
@@ -20,6 +20,24 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  if (century) {
```

Comment:
Is it okay to use 'if' in this case? Or are they totally forbidden in this task?

Reply:
No, it's not okay. Yes, these are forbidden here
```suggestion
```